drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/RightPanelView.java

```diff
@@ -16,26 +16,35 @@
 
 package org.drools.workbench.screens.scenariosimulation.client.rightpanel;
 
+import java.util.Map;
+
+import com.google.gwt.dom.client.DivElement;
 import com.google.gwt.user.client.ui.IsWidget;
+import org.drools.workbench.screens.scenariosimulation.client.models.FactModelTree;
 import org.uberfire.client.mvp.HasPresenter;
 
 public interface RightPanelView
         extends IsWidget,
                 HasPresenter<RightPanelView.Presenter> {
 
-    void showEditorTab();
+    void clearInputSearch();
 
-    void hideCheatSheetTab();
+    void hideClearButton();
 
-    void showCheatSheetTab();
+    void showClearButton();
 
-    void hideEditorTab();
+    DivElement getListContainer();
 
     interface Presenter {
 
-        void onEditorTabActivated();
+        void clearSearch();
```

Comment:
To clearly point out these methods are used by the view please use the format onWhatActionIsHapening-format.

The View should not have actions on the Presenter, the Presenter should react on changes in the View.

I took a look at the methods, they are good, just the phrasing could be different.

Reply:
Fixed with
[DROOLS-2909] Renaming presenters' methods to follow common usage